src/Microsoft.DotNet.Interactive.Jupyter.Tests/ExecuteRequestHandlerTests.cs

```diff
@@ -455,27 +455,58 @@ public async Task Shows_not_supported_exception_when_stdin_not_allowed_and_passw
         }
 
         [Fact]
-        public void cell_language_can_be_pulled_from_metadata_when_present()
+        public void cell_kernel_name_can_be_pulled_from_dotnet_metadata_when_present()
         {
             var metaData = new Dictionary<string, object>
             {
-                { "dotnet_interactive", new InputCellMetadata ("fsharp") }
+                { "dotnet_interactive", new InputCellMetadata(language: "fsharp") }
```

Comment:
```suggestion
                { "dotnet_interactive", new InputCellMetadata(language: "F#") }
```

Reply:
It's important to use different values for the kernel and language names. Otherwise, it's easy to see how the tests can be false positives. They'll also communicate the wrong intent.

Reply:
This is a case where we were calling the field `languageName` but we were actually writing the kernel name.

Reply:
It sounds like we're overloading the concept of language name then, if `fsharp` and `F#` are both valid "language names" at various points. How should we disambiguate this?